.github/workflows/main.yml

```diff
@@ -30,4 +30,4 @@ jobs:
       run: yarn install
 
     - run: yarn lint
-    - run: yarn test
+    # - run: yarn test
```

Comment:
The problem that goes away from this change seems fundamental. I think we should try to fix it before merging this.

Reply:
I think what's happening is that the module system is looking for a `.js` file when it attempts to import `@/types`. If I change that to `@/types.ts`, then I get a different error, related to Jest not knowing how to run Babel on a typescript file.

I think you need to add TS support to the Vue project itself. I am not quite sure how to do that. Look into it a bit, and I'll consult some experts as well.

Reply:
Woohoo! With @subdavis's help, I was able to get the tests running (and passing)!